ext/drwrap/drwrap.c

```diff
@@ -2782,6 +2795,14 @@ drwrap_get_retaddr_if_sentinel(void *drcontext, INOUT app_pc *possibly_sentinel)
     /* If we see the sentinel, we must be inside a wrapped function. */
     ASSERT(pt != NULL && pt->wrap_level >= 0, "Invalid drwrap state.");
     *possibly_sentinel = pt->retaddr[pt->wrap_level];
+    /* In case of a tailcall, the return address has already been replaced by
```

Comment:
Wait: I thought if we replaced in the tailcall entry, we did not need this?  This can be removed, right?  Replaced by an assert that the value is not the sentinel?

Reply:
done.